application.go

```diff
@@ -36,23 +37,23 @@ type Applications struct {
 // Application is the definition for an application in marathon
 type Application struct {
 	ID                    string              `json:"id,omitempty"`
-	Cmd                   string              `json:"cmd,omitempty"`
-	Args                  []string            `json:"args"`
-	Constraints           [][]string          `json:"constraints"`
+	Cmd                   *string             `json:"cmd,omitempty"`
+	Args                  *[]string           `json:"args,omitempty"`
```

Comment:
yeah, that's what I meant. However: I _think_ the `/bin/sh -c ${app.cmd}` is just the default executor which can be changed using the `executor` property and then an empty value could be semantically correct.